.github/release-drafter.yml

```diff
@@ -0,0 +1,28 @@
+
+name-template: 'Prebid $RESOLVED_VERSION Release'
+tag-template: '$RESOLVED_VERSION'
+categories:
+  - title: '🚀 New Features'
+    label: 'feature'
+  - title: '🧰 Maintenance'
+    label: 'chore'    
```

Comment:
a lot of the PRs that we end up putting in "Maintenance" are adapters/modules making small modifications to their code. Yes, you could argue it's a "chore", but that has a negative connotation. Could we instead (or in addition) just have a label "maintenance"?

Also...should we scope these labels at all so it's clear that these are for release drafter? something like `release-feature`, `release-bugfix` or something like that?

Reply:
Using maintenance sounds good and I agree the "chore" label has negative connotations. This is the time to pick our choice of words. I'll change it. 

In general though, I think that the labels are useful broadly and would prefer not to scope them. Scoping the labels would imply that is their only role. If the PR is a feature, then it's a feature and while that is metadata for release drafter it should/could be used without release drafter for other purposes. In addition, RD can pull in any labels we choose into any number of buckets. I would think of RD conforming to our labeling scheme that we use to label the type of PR, not the other direction.

If we really do want to scope them, then the verbosity of the label is important. I would prefer to frontload the wording so if they get truncated in a UI the more differentiating part is shown first instead of "release-...." over and over it would read "feature-...". It's unlikely, but possible.

Reply:
Makes total sense on not scoping the labels.